admin/homebrew.py

```diff
@@ -144,17 +131,39 @@ def main():
     If no command line argument is provided, use the standard release
     location for the indicated version.
     """
-    version = get_version()
-    if len(sys.argv) < 2:
+    logging.basicConfig(level=logging.INFO)
+
+    parser = argparse.ArgumentParser(description='Process some integers.')
+    parser.add_argument(
+        '--flocker-version', help='a version number for the Homebrew recipe')
+    parser.add_argument(
+        '--sdist', help='source distribution')
```

Comment:
Likewise, probably worth giving the help description as something like "URL of source distribution"